packages/devextreme/js/__internal/grids/grid_core/data_controller/m_data_controller.ts

```diff
@@ -1602,6 +1602,12 @@ export class DataController extends ControllerWithDataMixin {
   totalCount() {
     return this._dataSource ? this._dataSource?.totalCount() : 0;
   }
+
+  hasLoadOperation(): boolean {
+    const operationTypes = this._dataSource?.operationTypes() || {};
```

Comment:
Let's change it to the ```??``` if possible
We have the eslint rule that forces ```??``` instead of the ```||```

```suggestion
    const operationTypes = this._dataSource?.operationTypes() ?? {};
```